### ISSUE TITLE
allow column (and potentially other) ranges in bibliography

### ISSUE THREAD

**Issue by adam3smith** [citation-style-language/zotero-bits#62]
In some works, particularly in theology, column ranges rather than page ranges are given in the bibliography. While we do have columns and many other locators as locators for citations, this isn't currently possible - neither is there a CSL variable nor is there a Zotero field.
https://forums.zotero.org/discussion/31951/column-numbers-instead-of-page-numbers-in-bibliography/#Item_9
e.g.
Orth, F., art. « Schwein », PW II A/1 (1921), col. 801-815.
from http://www.sources-chretiennes.mom.fr/upload/doc/Bibliographie%20porcine.pdf
or 
Rentsch, Thomas / Cloeren, H. J., Art. Metaphysikkritik, in: Historisches Wörterbuch der Philosophie, Bd. 5, Basel-Darmstadt 1980, Sp. 1280–1294.
from http://books.google.com/books?id=fpgV5xtlT0oC&pg=PA708#v=onepage&q&f=false

**Comment by sarcanon** [comment]
Any movement on this ticket? I, too, would greatly appreciate having the ability to cite references that uses columns and folios, rather than merely pages. Very common in encyclopedic works from German sources.

**Comment by bwiernik** [comment]
My proposal:

Change `page` to have a type just like `locator`. All types that are valid for `locator` can be valid for `page`. If no type is supplied, assume that the type is `"page"`. 

Then, also adopt citeproc-js's system for multiple locator/page types by permitting a comma-delimited list with multiple locators differentiated by their label or abbreviation.

So, for example, to cite an item with a page and a column, it would be entered in `page` as:
`"page": "123, col. 2"`

or:
`"page": "page 123, column 2"`

The processor is expected to split such lists into an array and apply formatting rules separately to each element according to its type (e.g., APA rules want "Chapter" but short labels for other types).

**Comment by denismaier** [comment]
Adding types to pages sounds good, also splitting them into an array.

**Comment by denismaier** [comment]
What do we do about this now? Can we address this in 1.1?

**Comment by bwiernik** [comment]
Yes, 1.1 would be a good time (it would be easy for citeproc-js at least to add because it would just be extending its `locator` logic to `page`).

**Comment by denismaier** [comment]
Ok. I'll add a new issue in the schema repo and link to this one. Can you add add the Zotero label?
